interface/patient_file/ccr_pending_approval.php

```diff
@@ -18,6 +18,7 @@
 require_once(dirname(__FILE__) . "/../../library/patient.inc");
 require_once(dirname(__FILE__) . "/../../library/parse_patient_xml.php");
 
+use OpenEMR\Core\Header;
 use OpenEMR\Common\Csrf\CsrfUtils;
```

Comment:
rec ordering the use statements alphabetically (super anal I know, but will avoid duplicates when have large number of use statements in future).

Reply:
We could just automate a way to fix the order of the use statements since that's hard to track.

Reply:
No need to expend energy automating this process and risk something odd happening. I generally fix these whenever see them, especially in scripts using lots of use calls.